packages/teams-js/src/public/pages.ts

```diff
@@ -542,7 +543,7 @@ export namespace pages {
     }
 
     /**
-     * Navigates back in the hosted app. See registerBackButtonHandler for more information on when
+     * Navigates back in the hosted application. See registerBackButtonHandler for more information on when
```

Comment:
for more information on when its appropriate to use this method _>
for notes on usage

Reply:
Thanks for the note, @erikadoyle! Should I add a link to pages.backStack.registerBackButtonHandler?

Reply:
Ooh good call!